src/app/item/item.component.html

```diff
@@ -0,0 +1,7 @@
+ <li *ngIf="item">
+    <div class="store--item-icon">
+        <img src="./../../assets/icons/{{item.id}}.svg">
```

Comment:
Shouldn't need to do relative paths likes thing. You can just do `assets/icons/{{item.id}}.svg`